skymp5-server/cpp/server_guest_lib/MpObjectReference.cpp

```diff
@@ -31,9 +31,19 @@ std::string MpObjectReference::CreatePropertyMessage(
 nlohmann::json MpObjectReference::PreparePropertyMessage(
   MpObjectReference* self, const char* name, const nlohmann::json& value)
 {
+  std::string baseRecordType;
+
+  auto& loader = self->GetParent()->GetEspm();
+  auto base = loader.GetBrowser().LookupById(GetBaseId());
+  if (base.rec) {
+    baseRecordType = base.rec->GetType().ToString();
+  }
+
   return nlohmann::json{ { "idx", self->GetIdx() },
                          { "t", MsgType::UpdateProperty },
                          { "propName", name },
+                         { "refrId", self->GetFormId() },
+                         { "baseRecordType", baseRecordType },
```

Comment:
Why do we need them? I see no usages in the client code

Reply:
This must have been fixes I pulled from another repository. I wanted to merge the fixes by @Pospelove as they weren't in the original repository. (Without those fixes the game stutters a lot.)